### ISSUE TITLE
Use of <details> <summary> to expand/collapse

### ISSUE THREAD

**Issue by haltersweb** [w3c/wai-policies-prototype#22]
I suggest not using details / summary for expanding and collapsing content.

This is not widely supported with screen reader / browser combos.

1. IE does not support details/summary at all. (http://caniuse.com/#feat=details) Since JAWS is a major screen reader which pairs with IE this would be an issue.
2. I tried it in NVDA and while the summary is announced as a button with the current state announced, when triggered it doesn't announce the change of state.
3. I tried it in VO on iOS and summary cannot be reached and engaged.
4. I tried it in VO on Mac and it doesn't work well with the screen reader and the instructions given by VO do not work if followed.

I would suggest using a regular button with aria-expanded and aria-owns.

Here is some extra info: http://haltersweb.github.io/Accessibility/accordion.html

**Comment by AndrewArch** [comment]
Thanks @haltersweb - we will onboard for implementation (noting this a rough prototype)

**Comment by maryjom** [comment]
I think this will be able to be closed once all of the list of detailed countries are removed from the overview table page.

**Comment by iamjolly** [comment]
We are no longer using the `<details>` and `<summary>` elements in this version of the policies MVP. I'm going to close it with big thanks to @haltersweb for raising that IE/Edge issue. If we reintroduce this show/hide feature, we'll implement it to have support for everything—either via polyfill or a different approach altogether.

**Comment by iamjolly** [comment]
Reopening. I was mistaken. We still had one `<details><summary>` section on the index page for the table (which can likely be removed. But I also just added the Terms back to a collapsible `<details>` section in the Intro of that page, so I'll port over the polyfill from the WAI-Tools repo. Once that's done and tested in IE10+ and Edge, we'll be able to close this one.

**Comment by iamjolly** [comment]
And this is now fixed. IE and Edge have support now for the expandable/collapsible details content.
